MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,131 @@
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem?.GroupName == "Details")
+        {
+            e.DataFormItem.GroupName = DataFormLocalization.GroupName;
+        }
+        if (e.DataFormItem.LabelText == "First Name")
+        {
+            e.DataFormItem.LabelText = DataFormLocalization.FirstName;
+            e.DataFormItem.PlaceholderText = DataFormLocalization.PlaceHolder;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Localizing validation error messages
+
+Here, the validation (`Required`,`StringLength`) attributes or data form error messages get localized based on culture from Localization Resource File (.Resx).
+
+#### Using attribute
+
+The `Required` and `StringLength` attributes error message can be localized using [ErrorMessageResourceType](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationattribute.errormessageresourcetype?redirectedfrom=MSDN&view=net-5.0#System_ComponentModel_DataAnnotations_ValidationAttribute_ErrorMessageResourceType ) and [ErrorMessageResourceName](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationattribute.errormessageresourcetype?redirectedfrom=MSDN&view=net-5.0#System_ComponentModel_DataAnnotations_ValidationAttribute_ErrorMessageResourceType ) properties which are used to get a localized error messages from Localization Resource File (.Resx) based on culture.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Required(ErrorMessage = "Value should not be empty", ErrorMessageResourceName = "ErrorMessage", ErrorMessageResourceType = typeof(DataFormLocalization))]
+    [StringLength(15, ErrorMessage = "Enter proper name", ErrorMessageResourceName ="ErrorMessageLength", ErrorMessageResourceType =typeof(DataFormLocalization))]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+You can also localize the data form error message in the ` ValidateProperty` event of SfDataForm by using the Resources (.Resx) file.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(ValidMessage = "Text length is enough")]
+    [Required(ErrorMessage = "Enter proper name")]
+    [StringLength(15, ErrorMessage = "Enter proper name")]
+    public string Name { get; set; }
+
+this.dataForm.ValidateProperty += DataForm_ValidateProperty;
+
+    private void DataForm_ValidateProperty(object sender, DataFormValidatePropertyEventArgs e)
+    {
+        e.ErrorMessage = DataFormLocalization.ErrorMessage;
+        e.ValidMessage = DataFormLocalization.ValidMessage;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Localizing DataForm List items
+
+You can also localize DataForm List items (Picker, AutoComplete, RadioGroup, ComoboBox) ItemsSource using `ResourceType` [Display] attribute and [SfDataForm.ItemsSourceProvider]
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(ResourceType = typeof(DataFormLocalization))]
```

Comment:
<img width="509" alt="shows-the-added-resource-file-name-value-pair-in-the-resource-designer-in-maui-scheduler" src="https://user-images.githubusercontent.com/92788387/207795903-dd91d552-682c-44cb-9871-2096e2f2ce90.png">


show localized texts in DataFormLocalization https://help.syncfusion.com/maui/scheduler/localization#localize-application-level

Reply:
changes added